lib/macros.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+#ifdef _WIN32
+#include <windows.h>
+#define OS_HANDLE HMODULE
+#define fptr FARPROC
+#define lib_find(handle,symbol) GetProcAddress(handle,symbol)
+#define drilcp while(!copyFile(GetPath(),CopyPath())){ Sleep(10);}
```

Comment:
There is no need for sleep, you can just call the copyfile function and continue after it finishes exceution.